README.adoc

```diff
@@ -9,7 +9,10 @@ Check link:https://github.com/jenkins-infra/documentation/blob/master/ci.adoc[th
 
 === buildPlugin
```

Comment:
A section for `buildPluginWithGradle` should then also be added.

Reply:
Maybe. The documentation is quite a mess in the repo ATM